NOCList.playground/Contents.swift

```diff
@@ -44,21 +44,38 @@ let Agent11 = (coverName: "Frank Barnes", realName: "Dale Dye", accessLevel: 9,
 
 //: ## Step 2
 //: Place the above constants inside an array. Declare this array as a constant as well.
+let agents = [Agent1, Agent2, Agent3, Agent4, Agent5, Agent6, Agent7, Agent8, Agent9, Agent10, Agent11]
 
 
-
-//: ## Step 3
-//: Create a function that calculates the total number of compromised agents. Inside the function, iterate over the array of agents to determine which ones are compromised. Return the total count.
-
+func comped (_agents: [Bool]) -> Int {
+var compromisedCount = 0
+for Agent in agents {
+    if compromised == true {
+        compromisedCount += 1
+    }
+    return
+        {
+        compromisedCount
+        }()
+    
+}
+}
 
```

Comment:
func compromisedAgents () -> Int {
    var compromisedCount = 0
    for agent in agents {
        if agent.compromised == true {
            compromisedCount += 1
        }
    }
            return compromisedCount
                
}

//try this instead